tests/integration_tests/pose_estimation_dataset_test.py

```diff
@@ -0,0 +1,36 @@
+import os
+import unittest
+
+import pkg_resources
+from hydra import initialize_config_dir, compose
+from hydra.core.global_hydra import GlobalHydra
+
+import super_gradients
+from super_gradients.common.environment.path_utils import normalize_path
+from super_gradients.training.dataloaders.dataloaders import _process_dataset_params
+from super_gradients.training.datasets.pose_estimation_datasets import COCOKeypointsDataset
+
+
+class PoseEstimationDatasetIntegrationTest(unittest.TestCase):
```

Comment:
nice, can we also add datalaoders instantiation to tests/unit_tests/datalaoder_factory_test.py ?